File: Scala/FileIO/src/Write.scala
import java.io.{BufferedWriter, FileWriter}
object Write {
  def main(args: Array[String]): Unit = {
    val out = new BufferedWriter(new FileWriter("test.txt"))
    var GB = 1*1024*1024*1024
    var size = 0
    while(GB!=0) {
      out.write('A')
      size+=1
      GB-=1
    }
    out.close()
    println("Size: "+size)
  }
}

File: Java/FileIO/src/Write.java
import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;

public class Write {

	public Write() {
		try {
			BufferedWriter out = new BufferedWriter(new FileWriter("test.txt"));
			long GB = 1*1024*1024*1024;
			while(GB--!=0) out.write('A');
			out.close();
		} catch (IOException e) {
			e.printStackTrace();
		}
		
	}

	public static void main(String[] args) {
		new Write();
	}

}
